lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -128,10 +129,6 @@ def call(env)
           # unless it has been already set by the underlying framework
           if status.to_s.start_with?('5') && request_span.status.zero?
             request_span.status = 1
-            # in any case we don't touch the stacktrace if it has been set
```

Comment:
Well, the code below has been removed because it was only a fallback "in case we don't have the exception" trying to fill the stack trace with something. It appears it was a bad idea, it confuses users, as the stack trace is only about... our code, which provides no insight for users to spot where the problem actually was. If the error was catched by another layer, we just do not have the information any more, period, so rather than report misleading info, report nothing.